services/userService.js

```diff
@@ -0,0 +1,61 @@
+const User = require("../models/user");
+
+async function createUser(data) {
+  try {
+    const user = await User.create(data);
+    return user;
+  } catch (error) {
+    throw error;
+  }
+}
+
+async function getUserById(id) {
+  try {
+    const user = await User.findByPk(id);
+    if (!user) {
+      throw new Error("User not found");
+    }
+    return user;
+  } catch (error) {
+    throw error;
+  }
+}
+
+async function getAllUsers() {
+  try {
```

Comment:
The code snippet only contains a `try` block without a corresponding `catch` block, which is not a complete error handling mechanism. It's better to include a `catch` block to handle any potential errors.